apps/script/aichatCloudWatch/alarms/alarmConfigurations.ts

```diff
@@ -0,0 +1,100 @@
+import {ComparisonOperator} from '@aws-sdk/client-cloudwatch';
+
+import modelDescriptions from '../../../static/aichat/modelDescriptions.json';
+
+import {
+  createExecutionJobMetricStat,
+  createOpenaiMetricStat,
+} from './alarmHelpers';
+
+const modelIds = modelDescriptions.map((model: {id: string}) => model.id);
+
+export const openaiLowSuccessRateConfiguration = {
+  AlarmName: 'test_script_genai_openai_low_success_rate',
+  AlarmDescription: 'test_script_genai_openai_low_success_rate',
+  ActionsEnabled: true,
+  OKActions: [],
+  AlarmActions: ['arn:aws:sns:us-east-1:475661607190:javabuilder-low-urgency'],
```

Comment:
These should notify an appropriate team/application based SNS topic.

Reply:
Thanks - this is just a placeholder for now in this draft PR.